mysql-test/suite/s3/suite.pm

```diff
@@ -4,5 +4,45 @@ package My::Suite::S3;
 
 return "Need S3 engine" unless $::mysqld_variables{'s3'} eq "ON" or $ENV{HA_S3_SO};
 
+if (!$ENV{'S3_HOST_NAME'})
+{
+  return "Environment variable S3_HOST_NAME need to be set";
+}
+
+if (!$ENV{'S3_BUCKET'})
+{
+  return "Environment variable S3_BUCKET need to be set";
+}
+
+if (!$ENV{'S3_REGION'})
+{
+  return "Environment variable S3_REGION need to be set";
+}
+
+if (!$ENV{'S3_ACCESS_KEY'})
+{
+  return "Environment variable S3_ACCESS_KEY need to be set";
+}
+
+if (!$ENV{'S3_SECRET_KEY'})
+{
+  return "Environment variable S3_SECRET_KEY need to be set";
+}
+
+if (!$ENV{'S3_PROTOCOL_VERSION'})
+{
+  $ENV{'S3_PROTOCOL_VERSION'} = "Auto";
+}
+
+if (!$ENV{'S3_PORT'})
+{
+  $ENV{'S3_PORT'} = 80;
```

Comment:
I've just updated the documentation, this default is `0` and its probably better if this does the same, less variables to set.